Vagrantfile

```diff
@@ -8,11 +8,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # Every Vagrant virtual environment requires a box image to base.  Use a 64bit
   # box from the latest Ubuntu release; if it does not exist, vagrant up will
   # fetch a box and name it properly.
-  config.vm.box = "saucy64"
-
-  # The url from where the 'config.vm.box' box will be fetched if it
-  # doesn't already exist on the user's system.
-  config.vm.box_url = "http://cloud-images.ubuntu.com/vagrant/saucy/current/saucy-server-cloudimg-amd64-vagrant-disk1.box"
+  config.vm.box = "ubuntu/trusty64"
```

Comment:
You said vivid in your description but this is trusty. Was the description incorrect or the code? :)